### PR TITLE
DOC-3815: Key concepts - time-windows

### PR DESCRIPTION
**Pull request by AaronForinton**

### CODE REVIEW

File: content/en/platform/corda/4.8/open-source/key-concepts-time-windows.md [hunk 1]
```diff
@@ -31,45 +31,40 @@ title: Time-windows
...
-* A transaction occurring at some point after the given time (e.g. after a maturity event)
-* A transaction occurring at any time before the given time (e.g. before a bankruptcy event)
-* A transaction occurring at some point roughly around the given time (e.g. on a specific day)
+* A transaction occurring at some point after the given time, such as after a maturity event.
+* A transaction occurring at any time before the given time, such as before a bankruptcy event.
+* A transaction occurring at some point roughly around the given time, such as on a specific day.
 
-If a time window needs to be converted to an absolute time (e.g. for display purposes), there is a utility method to
-calculate the mid point.
+If a time-window needs to be converted to an absolute time, such for display purposes, there is a utility method to
```
Comment (AaronForinton) [inline content/en/platform/corda/4.8/open-source/key-concepts-time-windows.md:63]:
```suggestion
If a time-window needs to be converted to an absolute time, such as for display purposes, there is a utility method to
```

File: content/en/platform/corda/4.8/open-source/key-concepts-time-windows.md [hunk 2]
```diff
@@ -31,45 +31,40 @@ title: Time-windows
...
+As long as the party is not required to commit to the associated transaction, it can choose to reveal this fact at some point in the future. As a result the notary
+timestamps and notarizes at the same time.
 
-There will never be exact clock synchronisation between the party creating the transaction and the notary.
-This is not only due to issues of physics and network latency, but also because between inserting the command and
-getting the notary to sign there may be many other steps (e.g. sending the transaction to other parties involved in the
-trade, requesting human sign-off…). Thus the time at which the transaction is sent for notarisation may be quite
-different to the time at which the transaction was created.
+Thus, the time at which the transaction is sent for notarization may be
+different to the transaction creation time. There will never be exact clock synchronization between the party creating the transaction and the notary for two reasons:
```
Comment (erinking-r3) [inline content/en/platform/corda/4.8/open-source/key-concepts-time-windows.md:42]: This reordering confused me a bit - it made it seem like the fact that the node can reveal the fact later affects the time perception between the node and notary, but I think it might be something more like this:
```suggestion
A node may not send a transaction to the notary right away—they might need to circulate the transaction to other nodes involved in the transaction, or request human sign-off. Even if the node sends it as soon as it's generated, the node's clock and the notary's clock will never be perfectly in sync due to latency and physics. This means that the timestamp on a transaction is usually different from the time it was created.  
```

Comment (erinking-r3) [reply content/en/platform/corda/4.8/open-source/key-concepts-time-windows.md:42]: I also think this paragraph belongs a bit further down — at this point it's not clear why we would care about a tiny lag, but later you learn that this is the reason for time-windows rather than exact times.

File: content/en/platform/corda/4.8/open-source/key-concepts-time-windows.md [hunk 3]
```diff
@@ -31,45 +31,40 @@ title: Time-windows
...
+* Issues of physics and network latency
+* Between inserting the command and getting the notary to sign there may be many other steps, such as sending the transaction to other parties involved in the
+trade, or requesting human sign-off.
 
 ## Time-windows
 
-For this reason, times in transactions are specified as time *windows*, not absolute times. In a distributed system
-there can never be “true time”, only an approximation of it. Time windows can be open-ended (i.e. specify only one of
-“before” and “after”) or they can be fully bounded.
```
Comment (erinking-r3) [inline content/en/platform/corda/4.8/open-source/key-concepts-time-windows.md:54]: This is the bit that needs to be in the 'time in a distributed system' section, above.

File: content/en/platform/corda/4.8/open-source/key-concepts-time-windows.md [hunk 4]
```diff
@@ -31,45 +31,40 @@ title: Time-windows
...
+Times in transactions are specified as time *windows*, not absolute times. Time-windows can be open-ended, and specify only
+“before” **or** “after”, or they can be fully bounded.
 
 {{< figure alt="time window" width=80% zoom="/en/images/time-window.gif" >}}
 
-In this way, we express the idea that the *true value* of the fact “the current time” is actually unknowable. Even when
-both a before and an after time are included, the transaction could have occurred at any point within that time-window.
+When both a before and an after time are included, the transaction could have occurred at any point within that time-window.
```
Comment (erinking-r3) [inline content/en/platform/corda/4.8/open-source/key-concepts-time-windows.md:54]: Can you rephrase this in the active voice?

File: content/en/platform/corda/4.8/open-source/key-concepts-time-windows.md [hunk 5]
```diff
@@ -31,45 +31,40 @@ title: Time-windows
...
 
-By creating a range that can be either closed or open at one end, we allow all of the following situations to be
+By creating a range that can be either closed or open at one end, we allow all the following situations to be
 modelled:
```
Comment (erinking-r3) [inline content/en/platform/corda/4.8/open-source/key-concepts-time-windows.md:57]:
```suggestion
Time windows let you represent transactions that follow different models, such as those that occur:
```
Some more style guide notes here: we don't use "we" in docs. It's also ideal to rephrase wordy phrases such as "allows you to" with "lets" and to eliminate necessary phrases like "the following".

For lists, it's better to use a phrase that applies to all the bullet points in the intro section so it's not repeated.

File: content/en/platform/corda/4.8/open-source/key-concepts-time-windows.md [hunk 6]
```diff
@@ -31,45 +31,40 @@ title: Time-windows
...
 
-* A transaction occurring at some point after the given time (e.g. after a maturity event)
-* A transaction occurring at any time before the given time (e.g. before a bankruptcy event)
-* A transaction occurring at some point roughly around the given time (e.g. on a specific day)
+* A transaction occurring at some point after the given time, such as after a maturity event.
+* A transaction occurring at any time before the given time, such as before a bankruptcy event.
+* A transaction occurring at some point roughly around the given time, such as on a specific day.
```
Comment (erinking-r3) [inline content/en/platform/corda/4.8/open-source/key-concepts-time-windows.md:61]:
```suggestion
* At some point after the given time, such as after a maturity event.
* At any time before the given time, such as before a bankruptcy event.
* Around the given time, such as on a specific day.
```

File: content/en/platform/corda/4.8/open-source/key-concepts-time-windows.md [hunk 7]
```diff
@@ -31,45 +31,40 @@ title: Time-windows
...
 
-If a time window needs to be converted to an absolute time (e.g. for display purposes), there is a utility method to
-calculate the mid point.
+If a time-window needs to be converted to an absolute time, such as for display purposes, there is a utility method to
+calculate the mid-point.
```
Comment (erinking-r3) [inline content/en/platform/corda/4.8/open-source/key-concepts-time-windows.md:64]:
```suggestion
If you need to convert a time window to an absolute time, such as for display purposes, you can use a utility method to calculate the midpoint.
```
suggest the active here

File: content/en/platform/corda/4.8/open-source/key-concepts-time-windows.md [hunk 8]
```diff
@@ -31,45 +31,40 @@ title: Time-windows
...
 
 {{< note >}}
-It is assumed that the time feed for a notary is GPS/NaviStar time as defined by the atomic
-clocks at the US Naval Observatory. This time feed is extremely accurate and available globally for free.
+It is assumed that the time feed for a notary is [GPS/NaviStar time as defined by the atomic
+clocks at the US Naval Observatory](https://www.usno.navy.mil/USNO/time/display-clocks/simpletime). This time feed is extremely accurate and available globally for free.
```
Comment (erinking-r3) [inline content/en/platform/corda/4.8/open-source/key-concepts-time-windows.md:68]:
```suggestion
Most notaries use [GPS/NaviStar time](https://www.usno.navy.mil/USNO/time/display-clocks/simpletime) as defined by the atomic clocks at the US Naval Observatory. This time feed is extremely accurate and available globally for free.
```

**Review by erinking-r3 — COMMENTED** [review]
Good work on this tricky topic. I left a few notes that will hopefully help make it easier for people to grasp.

**Review by erinking-r3 — APPROVED** [review]
LGTM
